src/test/regress/upgrade/generate_citus_tarballs.sh

```
@@ -0,0 +1,58 @@
+#!/bin/bash
+
```

Let's add `set -euxo pipefail`, so the script will fail fast.